tests/tasks/get_profile_stat.yml

```diff
@@ -1,7 +1,9 @@
 # SPDX-License-Identifier: BSD-3-Clause
 ---
-- name: Initialize NM profile exist flag
-  set_fact: lsr_net_profile_exists=false
+- name: Initialize NM profile exist and ansible_managed comment flag
+  set_fact:
+    lsr_net_profile_exists=false
+    lsr_net_profile_ansible_managed=false
```

Comment:
```yaml
set_fact:
  lsr_net_profile_exists: false
  lsr_net_profile_ansible_managed: false
```

Reply:
@richm, This is from set_fact doc.
```
The set_fact module takes ``key=value`` pairs or ``key: value``(YAML notation) as variables to set in the playbook scope. The 'key' is the resulting variable name and the value is, of course, the value of said variable.
You can create multiple variables at once, by supplying multiple pairs, but do NOT mix notations.
```
What'd be the preferable notation in `set_fact`? If there is just one line as `set_fact: lsr_net_profile_exists=true`, choose `=`; otherwise, the YAML notation?

Reply:
If the surrounding network role code uses the `=` style, then use it, to be consistent - otherwise, use the yaml style

Reply:
Both styles are found in the tests.
E.g., (in the main branch)
```
tests/tests_vlan_mtu_nm.yml
      set_fact:
        network_provider: nm

tests/playbooks/tasks/get_profile_stat.yml:  set_fact: lsr_net_profile_exists=false
tests/playbooks/tasks/get_profile_stat.yml:  set_fact: lsr_net_profile_exists=true
tests/playbooks/tasks/get_profile_stat.yml:  set_fact: lsr_net_profile_exists=true
```
I'm adding a new variable `lsr_net_profile_ansible_managed` with `set_fact` to tests/playbooks/tasks/get_profile_stat.yml, where `=` style was used. But I'd think it's ok to replace it with the YAML style.